lib/logflare/source/recent_logs_server.ex

```diff
@@ -229,26 +113,6 @@ defmodule Logflare.Source.RecentLogsServer do
     {:ok, current_cluster_inserts, current_inserts}
   end
 
-  def load_init_log_message(source_token) do
```

Comment:
This is useful to see in some cases, but not married to it. Can add it back later if we want.